src/scripts/main.js

```diff
@@ -1,5 +1,33 @@
 'use strict';
 
 document.addEventListener('click', (e) => {
-  // write code here
+  const targetDiv = e.target.closest('div');
```

Comment:
Ensure that `targetDiv` is not null before accessing its properties. The `closest` method can return null if no matching element is found.